diff -ru openmodelica_1.9.7~2-g3a02b55/OMEdit/qjson/CMakeLists.txt openmodelica_1.9.7~2-g3a02b55-patched/OMEdit/qjson/CMakeLists.txt
--- openmodelica_1.11.0~4-g67594c0/OMPlot/qwt/Makefile.unix.in	2018-09-07 11:43:10.000000000 +0200
+++ openmodelica_1.11.0~4-g67594c0-patched/OMPlot/qwt/Makefile.unix.in	2018-09-07 12:49:25.474488878 +0200
@@ -14,7 +14,7 @@ all: build
 
 Makefile: qwt.pro
 	@rm -f $@
-	$(QMAKE) QMAKE_CXX=@CXX@ qwt.pro
+	$(QMAKE) QMAKE_CXX=@CXX@ QMAKE_CXXFLAGS="@CXXFLAGS@" QMAKE_LINK="@CXX@" qwt.pro
 clean:
 	test ! -f Makefile || $(MAKE) -f Makefile clean
 	rm -rf build lib Makefile
